packages/cactus-plugin-ledger-connector-ethereum/src/main/json/openapi.json

```diff
@@ -0,0 +1,1164 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "Hyperledger Cacti Plugin - Connector Ethereum",
+    "description": "Can perform basic tasks on a Ethereum ledger",
+    "version": "0.0.1",
+    "license": {
+      "name": "Apache 2.0",
+      "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
```

Comment:
@outSH There are some new custom-checks in the CI that might start failing because of the version not matching the current release. Just a note for future reference so you know if it happens!

Reply:
Copied this comment to dependent PR to fix it there - https://github.com/hyperledger/cacti/pull/2581